.travis.yml

```diff
@@ -23,6 +23,11 @@ env:
     - TOXENV=py37-pluggymaster PYTEST_NO_COVERAGE=1
     - TOXENV=py37-freeze PYTEST_NO_COVERAGE=1
 
+matrix:
+  allow_failures:
+    - python: '3.8-dev'
+      env: TOXENV=py38
```

Comment:
different ordering than below, but it doesn't matter -- just something I noticed ;)